Documentation/upgrade.md

```diff
@@ -202,20 +214,45 @@ kubectl -n rook get pod ${API_POD_NAME} -o jsonpath='{.status.phase}{"\n"}{.spec
 Remember to verify the cluster health using the instructions found in the [health verification section](#health-verification).
 
 ### Monitors
-There are multiple monitor pods to upgrade and they are each individually managed by their own replica set.
-**For each** monitor's replica set, you will need to update the pod template spec's image version field to `rook/rook:v0.5.1`.
-For example, we can update the replica set for `mon0` with:
+#### Upgrading from 0.6.x or lower
+If upgrading from a version of Rook of 0.6.x, the upgrade of monitor pods is done differently. In 0.6.x or lower, monitor pods were being managed by a `replicaset`.
+To upgrade the monitor pods to the new version, we rely on the operator's monitor fail over feature.
+
+To begin the upgrade, you will need to delete each monitor's replica set one at a time. The operator will detect that the monitor is not in quorum and it will fail over the monitor by creating a new monitor pod.
+Since the operator has already been updated, the new monitor pod will be managed by a `deployment` resource and it will have the new image version. The new monitor pod will also have a new identity. So if you started
+with three monitor replica sets, the newly created monitor pod will have an identity of `rook-ceph-mon3`.
+
+For example, we can delete the replica set for `mon0` with:
 ```bash
-kubectl -n rook set image replicaset/rook-ceph-mon0 rook-ceph-mon=rook/rook:v0.5.1
+kubectl -n rook delete rs rook-ceph-mon0
 ```
 
-Once the replica set has been updated, we need to manually terminate the old pod which will trigger the replica set to create a new pod using the new version.
+**Note** that there is a timeout associated to failing over monitor pods. If using the default timeout, it will take 300 seconds (5 minutes) for the operator to fail over a monitor pod.
+This can be changed by setting the `ROOK_MON_OUT_TIMEOUT` variable in the rook-operator manifest.
```

Comment:
i think it's better not to mention the timeout setting. Then they would have to figure out how to restart the operator pod

Reply:
I think we need to otherwise the new monitor will take a long time to come up and the users will think there was a failure somewhere. Remember that these are manual steps. So you will have a human reading these while performing these actions.

Reply:
mentioning that it takes 300s to failover is a good thing. Telling them about how to change the timeout doesn't seem necessary.